subsys/pm/policy/Kconfig

```diff
@@ -2,6 +2,7 @@
 
 choice
 	prompt "Idle State Power Management Policy"
+	default PM_POLICY_APP if SOC_SERIES_EFR32BG22
```

Comment:
I do not think this is the correct location to do this.

Reply:
removed